chore: APP-265 add hint to drag file location

blushi opened the pull request:
## Description

https://regennetwork.atlassian.net/browse/APP-265

<!-- Add a description of the changes that this PR introduces and the files that
are the most critical to review. -->

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

<!-- This should include steps on how to test the fixes or features within the marketplace app and/or the website.
If applicable, this should also include links to the created/updated components on storybook. -->

From https://deploy-preview-2442--regen-marketplace.netlify.app/, log in and create a post for a project.
Upload a file and check that a hint has been added for the last location option "(drag map to choose location)"

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

blushi commented:
> @blushi I don't see the location hint.

Sorry I had only seen the hint below the radio label on figma, not the one on the map! I'll add it.

> This seems to happen when I upload a file with geolocation, then cancel out of uploading, then upload another file without geolocation.

Yeah separate bug, I can have a look to see if that can be fixed quickly as part of this, else I'll file a bug.

blushi commented:
@erikalogie could you have another look? I've also fixed the bug you mentioned.

r41ph commented on web-components/src/components/inputs/new/LocationPicker/LocationPicker.tsx:
```diff
@@ -129,8 +125,14 @@ export const LocationPicker = ({
         {/* We simply absolutely position the location pin at the center,
           without using a react-map-gl Marker, so the map moves around the pin */}
-        <div className="absolute top-[50%] left-[50%]">
+        <div className="absolute top-[50%] left-[50%] -ml-[18.5px] -mt-50">
```
Same here, while the impact is minimal too, there are built-in utilities for `left-[50%]` and `top-[50%]`:  `left-1/2` and `top-1/2`. Using built-in utilities helps reduce the size of the generated CSS.

r41ph commented on web-components/src/components/inputs/new/LocationPicker/LocationPicker.tsx:
```diff
@@ -88,15 +90,9 @@ export const LocationPicker = ({
-        cursor={disabled ? 'default' : undefined}
+        scrollZoom={{ around: 'center' }}
+        touchZoomRotate={{ around: 'center' }}
+        cursor={'default'}
```
Although the impact is probably minimal, this could be just a string.

erikalogie commented:
FYI I'm getting this error on the porfolio page, may be unrelated: 
![Screenshot 2024-08-28 at 5 05 07 PM](https://github.com/user-attachments/assets/06ac4ce3-1862-4875-a33e-e0b9be839452)

I'm getting this nonsense text for the hint:
![Screenshot 2024-08-28 at 5 06 12 PM](https://github.com/user-attachments/assets/c6d8b3e0-69ce-4315-85fd-22f77da1bc75)

blushi commented:
> FYI I'm getting this error on the porfolio page, may be unrelated: ![Screenshot 2024-08-28 at 5 05 07 PM]

this got fixed on dev separately, I've updated my branch to dev so this should be fixed here too.

> I'm getting this nonsense text for the hint

fixed

erikalogie commented:
LGTM! 

Just FYI the preview is not loading here, is there a bug filed for this already?: 
![Screenshot 2024-08-29 at 9 57 18 AM](https://github.com/user-attachments/assets/229f7926-7d6f-4f49-a773-6a52bd5d54f4)

blushi commented:
> LGTM!
> 
> Just FYI the preview is not loading here, is there a bug filed for this already?: ![Screenshot 2024-08-29 at 9 57 18 AM](https://private-user-images.githubusercontent.com/8460237/362717674-229f7926-7d6f-4f49-a773-6a52bd5d54f4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ5NDAzMzEsIm5iZiI6MTcyNDk0MDAzMSwicGF0aCI6Ii84NDYwMjM3LzM2MjcxNzY3NC0yMjlmNzkyNi03ZDZmLTRmNDktYTc3My02YTUyYmQ1ZDU0ZjQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MjlUMTQwMDMxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmY0OWI2YzhkNGM0YjRiODFjODdiZDcwOTJiNjFhNTY4MDE2ZGYxMTdjOWYwZGVjYzg1ZTJiZmY2NTVkZTUyMyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.1CxnvNbDGI0dGilf8Bgrm_HaBGOKWlIK3WBjueNBXL4)

I've modified some s3 permissions which could have an impact on this. I might need to update the server accordingly, I'll create an issue for that.
